source/jobs/data-verification/use-data-verification.txt

```diff
@@ -1,3 +1,115 @@
+.. _rm-use-data-verification:
+
 =====================
 Use Data Verification
-=====================
\ No newline at end of file
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Data verification runs as a separate process which is tightly
+coupled to a sync job. Data verification can be enabled for one-time 
+sync jobs. This page details how to enable and debug the 
+data verification process.
+
+Enable Data Verification
+------------------------
+
+You can enable data verification when :ref:`creating a sync job <rm-create-jobs>`
+on the :guilabel:`Data Migration` tab.
+
+On the :guilabel:`Migration Options` module, click the 
+:guilabel:`Verify migrated data` check box for your sync job.
+
+
+.. image:: /img/jobs/data-verification/jobs-verify-migrated-data.png
+   :alt: Enabled Data Verification
+
+.. note:: Data Verification Duration
+
+   The Data Verification process can take as long or longer than the data sync job.
+   You can click :guilabel:`Terminate` to cancel a Data Verification process.
+
+To view status and metadata information about the data verification process
+for a sync job, click the :guilabel:`SYNC JOB / DATA VERIFICATION` slider.
+
+.. image:: /img/jobs/data-verification/data-verification-click-slider.png
+   :alt: Click Data Verification Slider
+
+Data Verification Overview
+--------------------------
+
+On the :guilabel:`Data Verification` pane you can view the meta data 
+about a sync jobs verification process.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Syntax
+     - Description
+   * - Verification Status
+     - The state of the verification process.
+   * - Started
+     - The date time stamp that data verification process started.
+   * - Duration
+     - The total elapsed time of the data verification process.
+   * - Collections verified
+     - The total number of collections verified by the data verification process.
+   * - Documents verified
+     - The total number of documents verified by the data verification process.
+   * - Verification progress
+     - A real time percentage indicator of total work completed by the data verification process.
+
+
+Data Verification Statuses
+--------------------------
+
+A data verification process can be any of the following statuses:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Status
+     - Description
+   * - ABORTED
+     - A running data validation was unable to complete due to an error.
+   * - CANCELLED
+     - The data validation process was cancelled by clicking ``TERMINATE``.
+   * - CANCELLING
+     - The data validation process is being canceled.
+   * - FAILED
```

Comment:
(active voice)
s/b "...are listed on the Issues pane."

Reply:
Updated.